src/net.cpp

```diff
@@ -1953,29 +1952,29 @@ void CConnman::ThreadOpenAddedConnections()
 }
 
 // if successful, this moves the passed grant to the constructed node
-bool CConnman::OpenNetworkConnection(const CAddress& addrConnect, bool fCountFailure, CSemaphoreGrant *grantOutbound, const char *pszDest, bool fOneShot, bool fFeeler, bool manual_connection)
+void CConnman::OpenNetworkConnection(const CAddress& addrConnect, bool fCountFailure, CSemaphoreGrant *grantOutbound, const char *pszDest, bool fOneShot, bool fFeeler, bool manual_connection)
```

Comment:
Do we really want to remove the return argument here? I think it's useful for it to be able to signal failure, for example to log an error, even if we don't use that at the moment.

Reply:
Yes, this is actually a really big help for the libevent code. This needs to be a one-way function so that it can be run async, with failures optionally triggering callbacks.

The oneshot reinsertion was the only thing preventing making that change trivially.

Reply:
ok, makes sense then